develop/toolkit/interoperability/paraspell-xcm-sdk/transfer-tokens-between-parachains.md

```diff
@@ -0,0 +1,183 @@
+- Retrieve information regarding the transfer, along with fee estimates.
+- Submit the transaction.
+
+### Prerequisites
+
+- Knowledge of the [fundamentals of Polkadot](/polkadot-protocol/parachain-basics/){target=\_blank}
+- Basic understanding of [XCM](/develop/interoperability/intro-to-xcm/){target=\_blank}
+- Basic familiarity with JavaScript/TypeScript
+- Install [bun](https://bun.com/docs/installation){target=\_blank}, a JavaScript/TypeScript package manager
+
+## Initialize Your Project
+
+Create the project folder:
+
+```bash
+mkdir paraspell-transfer
+cd paraspell-transfer
+```
+
+Initialize the JavaScript project:
+
+```bash
+bun init -y
+```
+
+Install the required dependencies:
+
+```bash
+bun add @paraspell/sdk@11.3.2 polkadot-api@1.17.1 @polkadot-labs/hdkd-helpers@0.0.25 @polkadot-labs/hdkd@0.0.24
+```
+
+Now add the following setup code to `index.ts`:
+
+```ts title="index.ts"
+--8<-- 'code/develop/toolkit/interoperability/paraspell/index.ts:1:27'
+```
+
+Replace the `INSERT_YOUR_SEED_PHRASE` with the seed phrase from your Polkadot development account.
+
+Be sure to fund this account with some PAS tokens on Paseo's Asset Hub using the [Polkadot Faucet](https://faucet.polkadot.io/?parachain=1000){target=\_blank}.
+
+!!!warning "Security Warning"
+    Never commit your mnemonic phrase to production code. Use environment variables or secure key management systems.
+
+## Build a Token Transfer Transaction
+
+The next step is to build the transaction that you intend to execute.
+
+In this example, you will transfer 10 PAS tokens from Paseo's Asset Hub to Paseo's People Chain system parachain.
+
+Add the ParaSpell transaction code to your `index.ts` file:
+
+```ts title="index.ts"
+--8<-- 'code/develop/toolkit/interoperability/paraspell/index.ts:29:47'
+```
+
+Do not execute it just yet. You will perform a dry run of this transaction first to ensure it works as expected.
+
+## Perform a Dry Run
+
+Dry runs simulate the transaction without broadcasting it, allowing you to confirm success in advance.
+
+Add the following dry run code to your `index.ts` script:
+
+```ts title="index.ts"
+--8<-- 'code/develop/toolkit/interoperability/paraspell/index.ts:49:70'
+```
+Go ahead and run the script.
+
+```bash
+bun run index.ts
+```
+
+The result of the dry run will be similar to this:
+
+--8<-- 'code/develop/toolkit/interoperability/paraspell/dry-run-output.html'
+
+## Verify the Existential Deposit
+
+Check if the recipient account meets the [Existential Deposit (ED)](/polkadot-protocol/glossary/#existential-deposit){target=\_blank} requirement before sending by using [`verifyEdOnDestination`](https://paraspell.github.io/docs/sdk/xcmUtils.html#verify-ed-on-destination){target=\_blank}:
+
+```ts title="index.ts"
+--8<-- 'code/develop/toolkit/interoperability/paraspell/index.ts:72:88'
+```
+Comment out the `dryRunTransfer()` and execute the `verifyED()` by running:
```

Comment:
```suggestion
Comment out the `dryRunTransfer()` function so that it is not executed again. Then, execute the `verifyED()` by running the following command:
```